docs/source/library-user-guide/using-the-dataframe-api.md

```diff
@@ -19,4 +19,105 @@
 
 # Using the DataFrame API
 
-Coming Soon
+## What is a DataFrame
+
+`DataFrame` is a basic concept in `datafusion` and is only a thin wrapper over LogicalPlan.
```

Comment:
Perhaps it is worth explaining why we have both DataFrame and LogicalPlan since they are so similar. Maybe something like:

The main difference between a DataFrame and a LogicalPlan is that the DataFrame contains functionality for executing queries rather than just building plans.

Reply:
thank you! I have made it clear.